app/scripts/modules/core/entityTag/entityTags.read.service.ts

```diff
@@ -34,8 +60,39 @@ export class EntityTagsReader {
       tag.lastModifiedBy = metadata.lastModifiedBy;
     }
   }
+
+  private collateEntityIds(entityType: string, entityIds: string[]): string[] {
+    const baseUrlLength = `${this.settings.gateUrl}/tags?entityType=${entityType}&entityIds=`.length;
+    const maxIdGroupLength = get(this.settings, 'entityTags.maxUrlLength', 4000) - baseUrlLength;
+    const idGroups: string[] = [];
+
+    if (entityIds.join(',').length > maxIdGroupLength) {
```

Comment:
Without knowing the overhead of `.join`, does it make sense to cache this result since we do this op again later?